src/logger.h

```diff
@@ -34,19 +34,24 @@ extern "C" {
 #include "transport.h"
 #include "loggerPrivate.h"
 
-///@cond INTERNAL
+/// @cond INTERNAL
 extern LoggerData g_logger;
-///@endcond
+/// @endcond
 
 /**
  * @brief Initialize the Logging sub-system.
  * @param tx_cb The callback to use for sending messages when
- *   Logging_SendResponse() is called.
- * @param enabled true to enable the logger, false to disable.
+ *   Logging_SendResponse() is called. This can be overridden, see the note
+ *   below.
  * @param max_payload_size The maximum size of the payload to be passed to the
  *   TXFunction. Must be at least 2.
+ *
+ * @note If PIPELINE_TRANSPORT_TX is defined in system_pipeline.h, the macro
+ * will override the tx_cb argument.
+ *
+ * A Logger starts off in the disabled state.
```

Comment:
"Multiple adjacent \note commands will be joined into a single paragraph. Each note description will start on a new line."

If you don't want it as a note, can you move "A Logger starts off in the disabled state." up to below the @brief with a blank line in between, so it shows in the extended info, or use @post "Starts a paragraph where the postcondition of an entity can be described."